python/perf-kernels/flash-attention.py

```diff
@@ -200,7 +281,6 @@ def _attn_fwd_inner(
             encoded_softmax_block_ptr = tl.advance(encoded_softmax_block_ptr, (0, BLOCK_N))
     return acc, l_i, m_i
 
-
 @triton.jit
 def _attn_fwd(
```

Comment:
I would remove the leading `_` to indicate this kernel will be called outside this python file.

Can be done later.

Reply:
Fixed.